### PR TITLE
Add a CONTRIBUTING.md and a tag verification workflow

### PR DESCRIPTION
**Pull request by roehrich-hpe**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+We welcome contributions to dws
```
Comment (bdevcich-hpe) [inline CONTRIBUTING.md:1]: NNF

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+# Pull Requests
+
+This repository follows the standard GitHub fork+pull model.  We accept GitHub pull requests.
+
+Fork the project on GitHub, work in your fork and in branches, push these to your GitHub fork, and when ready, open up a GitHub pull request against `https://github.com/HewlettPackard/dws`
```
Comment (bdevcich-hpe) [inline CONTRIBUTING.md:7]: NNF

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+We welcome contributions to dws
+
+# Pull Requests
+
+This repository follows the standard GitHub fork+pull model.  We accept GitHub pull requests.
```
Comment (bdevcich-hpe) [inline CONTRIBUTING.md:5]: does it? 😉

Comment (roehrich-hpe) [reply CONTRIBUTING.md:5]: Yes.  That's how LLNL folks would submit any additions of their own.  The same thing they'd do for nnf-sos or nnf-dm.

**Comment by roehrich-hpe** [comment]
I've merged these fixes into [PR49](https://github.com/NearNodeFlash/NearNodeFlash.github.io/pull/49l)

I'll close this one.

**Pull request closed without merging**